codespell_lib/tests/test_dictionary.py

```diff
@@ -57,40 +60,42 @@ def test_dictionary_formatting(fname, in_aspell):
             err = err.lower()
             rep = rep.rstrip('\n')
             try:
-                _check_err_rep(err, rep, in_aspell, fname)
+                _check_err_rep(err, rep, in_aspell, fname, in_dictionary)
             except AssertionError as exp:
                 errors.append(str(exp).split('\n')[0])
     if len(errors):
         raise AssertionError('\n' + '\n'.join(errors))
 
 
-def _check_aspell(phrase, msg, in_aspell, fname):
-    if speller is None:
+def _check_aspell(phrase, msg, in_aspell, fname, languages):
+    if not spellers:  # if no spellcheckers exist
         return  # cannot check
     if in_aspell is None:
         return  # don't check
     if ' ' in phrase:
         for word in phrase.split():
-            _check_aspell(word, msg, in_aspell, fname)
+            _check_aspell(word, msg, in_aspell, fname, languages)
         return  # stop normal checking as we've done each word above
-    this_in_aspell = speller.check(
-        phrase.encode(speller.ConfigKeys()['encoding'][1]))
-    end = 'be in aspell for dictionary %s' % (fname,)
+    this_in_aspell = any(spellers[lang].check(phrase.encode(
+        spellers[lang].ConfigKeys()['encoding'][1])) for lang in languages)
+    end = 'be in aspell dictionaries (%s) for dictionary %s' % (
+        ', '.join(languages), fname,)
```

Comment:
The comma _after_ `fname` isn't necessary?

Reply:
deleted

Reply:
It's not necessary but it's not unnecessary either, it's something this project does so being consistent would be good. @larsoner explained why to me some time back.

Reply:
In the old version of this line:
```python
end = 'be in aspell for dictionary %s' % (fname,)
```
the comma _was_ required, in order to make `(fname,)` a tuple, but in the new version of the line:
```python
end = 'be in aspell dictionaries (%s) for dictionary %s' % (
        ', '.join(languages), fname,)
 ```
the trailing comma isn't needed, because `(', '.join(languages), fname)` is already a tuple.

Reply:
consistency sounds good. any rule to follow?

Reply:
https://github.com/codespell-project/codespell/pull/1142/files/dd4fa4d6b6173825246b8071821c90f1dd70b7d8#diff-03e414d16771e56e61ea094c97b8871ce002a9fe2615ee2ace1cc227fc03fb87

Reply:
If I see it correctly, we have no commas at the majority of tuples. 
Shall I now introduce it here again/everywhere? Or do we want to put the discussion/fix in a new pull request. 
Except of the comma, the PR should be mergeable.

Reply:
Apologies, re-reading @larsoner 's comment, it was just about commas at the end of single item tuples as @lurch said.